HW7.py

```diff
@@ -0,0 +1,57 @@
+# Task 1 Time
+
+motomin1 = 240
+motomin2 = 808
+currenthours1 = motomin1 // 60
+currenthours2 = motomin2 // 60
+currentmin1 = motomin1 % 60
+currentmin2 = motomin2 % 60
+# print(f'{currenthours1:02d}:{currentmin1:02d}')
+# print(f'{currenthours2:02d}:{currentmin2:02d}')
+
+numb1hour1 = currenthours1 // 10
+numb1hour2 = currenthours1 % 10
+numb1min1 = currentmin1 // 10
+numb1min2 = currentmin1 % 10
+
+numb2hour1 = currenthours2 // 10
+numb2hour2 = currenthours2 % 10
+numb2min1 = currentmin2 // 10
+numb2min2 = currentmin2 % 10
```

Comment:
Can be improved, since you are not using those variables:
```suggestion
numbsum2 = (currenthours2 // 10) + (currenthours2 % 10) + (currentmin2 // 10) + (currentmin2 % 10)
```